src/web/styles/tables.scss

```diff
@@ -36,4 +36,50 @@ table {
       background-color: var(--theme-background-row);
     }
   }
+
+  tr {
+    .tristate-checkbox {
+      margin-right: 16px;
+    }
+
+    button[role='checkbox'] {
+      width: 21px;
+      height: 21px;
+    }
+
+    .input-container {
+      height: 26px;
+    }
+  }
+}
+
+.no-table-data-container {
+  margin-top: 15px;
+  border: 1px solid var(--theme-border);
+  border-radius: 5px;
+  padding: 33px 24px;
+  display: flex;
+  align-items: center;
+  gap: 20px;
+
+  .table-data-text {
+    margin-left: 33px;
+
+    h2 {
+      margin-top: 0;
+    }
+  }
+}
```

Comment:
should this css block definitely be in `tables.scss`? Should SharingPermissionsTable, BusinessContactsTable and ParticipantRequestsTable use this same className?